### DISCUSSION TITLE
Milvus Backup+Restore To different type of schema

### DISCUSSION THREAD

**Discussion by daxesh020500** [milvus-io/milvus#35940] [Q&A and General discussion] [answered]
Does the Milvus backup and restore feature only support the collection to have same schema?

For example, I have two collections, every other fields have same name and type, just the vector fields are different

collection v1:FloatVector(512)
collection v2:Float16Vector(512)

So is it possible to copy the data from v1 collection and import it into v2 collection?

using mivlus v2.4.4.

Thank you.

**Comment by yhmo** [comment, marked as answer]
Sorry for missing this question.
The milvus-backup tool requires the schema of two collections must be equal, including the field name, field type, etc.
FloatVector and Float16Vector are different field types.

**Comment by xiaofan-luan** [comment]
I think the best way is to run a spark job to conver the data into new format and reinsert
